docs/src/zh/Development/Development-Guide.md

```diff
@@ -0,0 +1,452 @@
+        ```
+
+  - 第二种方法
+    * 把下方内容复制到一个文本文件中，文本名为 ```key.asc```
+
+        * 内容如下：
+            ```
+            -----BEGIN PGP PUBLIC KEY BLOCK-----
+            Version: GnuPG v2
+            ...
+            -----END PGP PUBLIC KEY BLOCK-----
+            ```
+
+            导入 RM 的公钥到自己电脑
+
+            ```
+            gpg2 --import key.asc
+  
+            ```
+
+- 4.验证源码发布版
+
+  * 验证是否有 NOTICE、LICENSE，以及内容是否正确。
+
+  * 验证 README、RELEASE_NOTES
+
+  * 验证 header
+
+    ```
+    mvn -B apache-rat:check
+    ```
+
+  * 验证签名和哈希值
+
+    ```
+    gpg2 --verify apache-iotdb-0.12.0-source-release.zip.asc apache-iotdb-0.12.0-source-release.zip
+
+    出现 Good Singnature 
+
+    shasum -a512 apache-iotdb-0.12.0-source-release.zip
+
+    和对应的 .sha512 对比，一样就可以。
+    ```
+
+  * 验证编译
+
+    ```
+    mvnw install
+
+    应该最后全 SUCCESS
+    ```
+- 5.验证二进制发布版
+
+    * 验证是否有 NOTICE、LICENSE，以及内容是否正确。
+
+    * 验证 README、RELEASE_NOTES
+
+    * 验证签名和哈希值
+
+        ```
+        gpg2 --verify apache-iotdb-0.12.0-bin.zip.asc apache-iotdb-0.12.0-bin.zip
+
+        出现 Good Singnature 
+
+        shasum -a512 apache-iotdb-0.12.0-bin.zip
+
+        和对应的 .sha512 对比，一样就可以。
+        ```
+
+    * 验证是否能启动以及示例语句是否正确执行
+
+        ```
+        nohup ./sbin/start-server.sh >/dev/null 2>&1 &
+
+        ./sbin/start-cli.sh
+
+        CREATE DATABASE root.turbine;
+        CREATE TIMESERIES root.turbine.d1.s0 WITH DATATYPE=DOUBLE, ENCODING=GORILLA;
+        insert into root.turbine.d1(timestamp,s0) values(1,1);
+        insert into root.turbine.d1(timestamp,s0) values(2,2);
+        insert into root.turbine.d1(timestamp,s0) values(3,3);
+        select * from root.**;
+
+        打印如下内容：
+        +-----------------------------------+------------------+
+        |                               Time|root.turbine.d1.s0|
+        +-----------------------------------+------------------+
+        |      1970-01-01T08:00:00.001+08:00|               1.0|
+        |      1970-01-01T08:00:00.002+08:00|               2.0|
+        |      1970-01-01T08:00:00.003+08:00|               3.0|
+        +-----------------------------------+------------------+
+
+        ```
+- 6.回复邮件
+
+    验证通过之后可以发邮件了
+
+    ```
+    Hi,
+
+    +1 (PMC could binding)
+
+    The source release:
+    LICENSE and NOTICE [ok]
+    signatures and hashes [ok]
+    All files have ASF header [ok]
+    could compile from source: ./mvnw clean install [ok]
+
+    The binary distribution:
+    LICENSE and NOTICE [ok]
+    signatures and hashes [ok]
+    Could run with the following statements [ok]
+
+    CREATE DATABASE root.turbine;
+    CREATE TIMESERIES root.turbine.d1.s0 WITH DATATYPE=DOUBLE, ENCODING=GORILLA;
+    insert into root.turbine.d1(timestamp,s0) values(1,1);
+    insert into root.turbine.d1(timestamp,s0) values(2,2);
+    insert into root.turbine.d1(timestamp,s0) values(3,3);
+    select * from root.**;
+
+    Thanks,
+    xxx
+    ```
```

Comment:
```suggestion

    ```
    Hi,

    +1 (PMC could binding)

    The source release:
    LICENSE and NOTICE [ok]
    signatures and hashes [ok]
    All files have ASF header [ok]
    could compile from source: ./mvnw clean install [ok]

    The binary distribution:
    LICENSE and NOTICE [ok]
    signatures and hashes [ok]

    Thanks,
    xxx
    ```
```